test/e2e/container_probe.go

```diff
@@ -75,7 +75,7 @@ var _ = Describe("Probing container", func() {
 		p, err := podClient.Create(makePodSpec(probe.withFailing().build(), nil))
 		expectNoError(err)
 
-		err = wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		err = wait.Poll(poll, 120*time.Second, func() (bool, error) {
```

Comment:
Done. The test doesn't correctly test the initial delay at all given that it also includes the pod startup time.